### DISCUSSION TITLE
Use the latest dev state of Dioxus from git ?

### DISCUSSION THREAD

**Discussion by ThomasCartier** [DioxusLabs/dioxus#2668] [Q&A] [answered]
Hi,
How would one use the latest iteration of Dioxus for fullstack dev ?
Many things have been implemented and I would like to test the latest main branch without waiting for the next stable release.
`cargo install --git https://github.com/DioxusLabs/dioxus dioxus-cli` don't seem to cut it.
Thanks,

**Comment by philibe** [comment, marked as answer]
Have you tried:
- `cargo install --git https://github.com/DioxusLabs/dioxus dioxus-cli --tag v0.5.2` 
     +  [v0.5.2](https://github.com/DioxusLabs/dioxus/releases/tag/v0.5.2) (April 5, 2024)
- `cargo install --git https://github.com/DioxusLabs/dioxus dioxus-cli --rev ACommitToChoose` 
     +  4 days ago

https://manpages.ubuntu.com/manpages/focal/en/man1/cargo-install.1.html

**Reply by ThomasCartier** [reply]
Thanks philibe, it worked
